track/templates/en/help.html

```diff
@@ -1,14 +1,19 @@
 {% extends "en/layout-en.html" %}
 
-{% block title %}Feedback{% endblock %}
+{% block title %}Get help{% endblock %}
 {% block pageid_en %}help{% endblock %}
 {% block pageid_fr %}aide{% endblock %}
 
 {% block content %}
 <section id="main-content" class="flex-1">
 	<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
-		<h1>Feedback</h1>
-		<p>For inquiries, questions, or comments about this dashboard or implementing HTTPS, please send an email to <a href="mailto:zzTBSCybers@tbs-sct.gc.ca" class="text-https-blue hover:text-black font-bold">TBS Cyber Security</a> or join the <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">HTTPS Everywhere group on GCconnex</a>.</p>
+		<h1>Get help</h1>
+		<h2 class="font-bold mb-4">General public</h2>
+		<p>For interpretation of any aspect of Implementing HTTPS for Secure Web Connections:   Information Technology Policy Implementation Notice (ITPIN), contact Treasury Board of Canada Secretariat through Public Enquiries.</p>
+		<h2 class="font-bold mt-4 mb-4">Government of Canada employees</h2>
+		<p>Individuals at departments should contact their departmental information technology group for any questions regarding this ITPIN. Individuals can also join the <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">HTTPS Everywhere group on GCconnex</a>, or the #HTTPSEverywhere channel on GCcollab Message.</p>
+		<p>Individuals from a departmental information technology group may contact the <a href="mailto:zzTBSCybers@tbs-sct.gc.ca" class="text-https-blue hover:text-black font-bold">TBS Cyber Security mailbox</a> for interpretations of this ITPIN.</p>
```

Comment:
whether or not `mailbox` is included in the `<a>` tag should be consistent with line 16